deta/base.py

```diff
@@ -99,12 +106,23 @@ def __init__(self, drive_name:str=None, project_key:str=None, project_id:str=Non
         self.name = drive_name
         self.project_key = project_key
         self.project_id = project_id
-        host = host or os.getenv("DETA_DRIVE_HOST") or "drive.deta.sh"
-        self.client = http.client.HTTPSConnection(host, timeout=3)
+        host = host or os.getenv("DETA_DRIVE_HOST") or "localhost:9015" # "drive.deta.sh"
+        self.client = http.client.HTTPConnection(host, timeout=300)
         self.base_path = "/v1/{0}/{1}".format(self.project_id, self.name)
         self.util = Util()
 
-
+    def get(self, name:str=None) -> typing.Optional[BufferedIOBase]:
+        assert name, "Please provide the name of the file to get."
+        code,res = self._request(f"/files/download?name={name}", "GET")
```

Comment:
`_request` reads the response, we shouldn't do that, we should just return the response before reading so that the user can read it how they want (chunked)

Reply:
Gotcha, thanks!